### DISCUSSION TITLE
Export pt to ptl not working ??

### DISCUSSION THREAD

**Discussion by danurrahmajati** [ultralytics/yolov5#6704] [Q&A]
I was trying to make a custom data train yolov5n with google colab. I want to try android implementation according to this pythorch demo https://github.com/pytorch/android-demo-app/tree/master/ObjectDetection

Then test the model with my computer and work well. Because i want to try object detection with an android device, I need to convert the pt format to ptl.

I am using export.py to get compatible weight pythorch lite using this code 

`python export.py --weights runs/train/exp/weights/best.pt --include torchscript`

The export process run without error. When I check with neutron, here the model looks like 

![image](https://user-images.githubusercontent.com/8369861/154803912-78a7c874-2306-418a-ad66-13b743d63c04.png)

there is a missing connection, when I try to use it for an android device, it failed to detect objects, bounding box appears at the top of the picture. 

Using google collab or my local machine gets the same result. Did I miss some setup?

**Comment by delongwu** [comment]
I also face your problem, but I had solved it . First retrain your model , then you must change something like this, please carefully check.

https://github.com/pytorch/android-demo-app/tree/master/ObjectDetection

and then you must change this code , like this 
https://github.com/pytorch/android-demo-app/issues/258
